web/pages/homepage.rb

```diff
@@ -3,6 +3,11 @@
 module Web
   module Pages
     class HomePage < BasePage
+      element :goods_sections, 'rz-goods-sections.goods-sections'
+
+      def wait_for_page_load
+        has_goods_sections?
+      end
```

Comment:
Want to clarify, this section is available on each page of the program or only available at the home page?

Reply:
This section is available only on the home page.